### PR TITLE
Update OCP4 Social IdP example setup in the latest docs

### LINKED ISSUE #22159: Update OCP4 Social IdP example setup in the latest docs
### Description

Among prerequisites of the [OCP4 Social IdP example setup](https://www.keycloak.org/docs/latest/server_admin/index.html#openshift-4), the `X509_CA_BUNDLE` variable In the latest version docs.
This is probably a leftover from the WF dist docs, as the variable isn't supported in the Quarkus based container image [1][2].

[1] https://www.keycloak.org/server/containers
[2] https://github.com/keycloak/keycloak/tree/main/quarkus/container

### PR DESCRIPTION
**Pull request by Pepo48**
* improved openshift.adoc

Closes #22159

### CODE REVIEW

**Comment by Pepo48** [comment]
@tondrusk PTAL, thanks.

File: docs/documentation/server_admin/topics/identity-broker/social/openshift.adoc [hunk 1]
```diff
@@ -38,36 +38,35 @@ grantMethod: prompt <4>
...
-. Installation of https://stedolan.github.io/jq/[jq].
-. `X509_CA_BUNDLE` configured in the container and set to `/var/run/secrets/kubernetes.io/serviceaccount/ca.crt`.
+. A certificate of the OpenShift 4 instance stored in the Keycloak Truststore.
+. A Keycloak server configured in order to use the truststore. For more information, see the https://www.keycloak.org/server/keycloak-truststore[Configuring a Truststore] {section}.
 
 .Procedure
-. Run the following command on the command line and note the OpenShift 4 API URL output.
+. Click *Clients* in the {project_name} menu.
+. Click *Identity Providers* in the {project_name} menu.
+. Enter your *Client ID*, *Client Secret*, *Valid Redirect URIs* and *Web Origins*.
```
Comment (vmuzikar) [inline docs/documentation/server_admin/topics/identity-broker/social/openshift.adoc:47]: I'm not sure I follow. :) Why to click Clients and create a Client? There's no Client at KC side. KC needs to have IdP, Client is created in OpenShift.

File: docs/documentation/server_admin/topics/identity-broker/social/openshift.adoc [hunk 2]
```diff
@@ -38,36 +38,35 @@ grantMethod: prompt <4>
...
-----
-curl -s -k -H "Authorization: Bearer $(oc whoami -t)" \https://<openshift-user-facing-api-url>/apis/config.openshift.io/v1/infrastructures/cluster | jq ".status.apiServerURL"
-----
+.Create client
+image:images/openshift-4-create-client.png[Create Client]
 +
-. Click *Identity Providers* in the {project_name} menu.
-. From the `Add provider` list, select `Openshift`.
+. From the `Social` section, select `Openshift v4` tile.
+. Enter the *Client ID* and *Client Secret* you used in the previous step and in the *Base URL* field, enter the API URL of your OpenShift 4 instance. Additionally, you can copy the *Redirect URI* to your clipboard.
```
Comment (vmuzikar) [inline docs/documentation/server_admin/topics/identity-broker/social/openshift.adoc:53]: See above. Client exists only in OpenShift, not in KC.

**Review by vmuzikar — CHANGES_REQUESTED** [review]
@Pepo48 Thank you, added a few comments.

**Comment by Pepo48** [comment]
> @Pepo48 Thank you, added a few comments.

ou're right, I accidentally mixed two use-cases, my bad. Thanks for the review.

**Review by vmuzikar — APPROVED** [review]
@Pepo48 LGTM, thanks!

@ahus1 Do you want to review as well?

**Comment by ahus1** [comment]
@vmuzikar - while I can get my hands on an OpenShift instance to test this, this part is out of scope for both store and cross-dc, and I have never used it this way. So I choose not to review it. I hope you understand. Merging this as I see you've already reviewed it.

**Comment by vmuzikar** [comment]
@ahus1 No problem. I pinged you just in case, as you've been keeping an eye on docs recently. ;)

**Pull request merged**